packages/models-library/src/models_library/function_service/demo_units.py

```diff
@@ -171,6 +185,13 @@ def build_input(schema):
                     "type": "number",
                 }
             ),
+            "radiation": build_input(
```

Comment:
NOTE: to make the test pass, just change the title of this entry to something different to ``Radiation``